### DISCUSSION TITLE
Option to KEEP Bluetooth off after Mac’s closed?

### DISCUSSION THREAD

**Discussion by CharlesMeyerson** [odlp/bluesnooze#40] [Ideas]
This is a great improvement—turning Bluetooth off when the MacBook’s closed has prolonged my battery life substantially. And I was happy to buy you a coffee!

But even better would be if Bluetooth didn’t automatically turn back _on_ when I open the Mac. An option to “turn off when closed and leave off” would be excellent.

**Comment by valoricDe** [comment]
Related right? https://github.com/odlp/bluesnooze/pull/16

**Comment by cheshi-mantu** [comment]
> This is a great improvement—turning Bluetooth off when the MacBook’s closed has prolonged my battery life substantially. And I was happy to buy you a coffee!
> 
> But even better would be if Bluetooth didn’t automatically turn back _on_ when I open the Mac. An option to “turn off when closed and leave off” would be excellent.

there is such option, just remove the app et voila!

![image](https://github.com/user-attachments/assets/e9101954-3896-46d9-85f9-e56e7593f7bf)

**Comment by CharlesMeyerson** [comment]
Can you elaborate a bit, with step-by-step instructions for dummies (like
me)?

**Reply by valoricDe** [reply]
I think @cheshi-mantu text was intended as a dumb joke. But I am not 100 percent sure xD

**Comment by CharlesMeyerson** [comment]
Gullible me!
